### DISCUSSION TITLE
Preview Environments for GitLab

### DISCUSSION THREAD

**Discussion by echarrod** [encoredev/encore#1450] [Suggestions]
I see in the https://encore.dev/docs/deploy/environments#preview-environments section it says:
> When you [connect your application to GitHub](https://encore.dev/docs/how-to/github), Encore will automatically provision ephemeral Preview Environments for each Pull Request.

Is there anything on the roadmap to allow support for GitLab too? 🙏

**Comment by marcuskohlberg** [comment]
Not currently, but we can investigate to see if the scope is small enough that we get it sorted in the near term.
Is this a blocker for you currently?
